### PR TITLE
Upgrade to play 2.5

### PR DESCRIPTION
**Pull request by svillafe**
Upgrade to play 2.5 

### Why do we need this? 
To be able to use the latest version of membership-common.

cc: @rupertbates

### CODE REVIEW

File: membership-attribute-service/app/filters/LogRequestsFilter.scala [hunk 1]
```diff
@@ -1,28 +0,0 @@
-package filters
-
-import play.api.Logger
-import play.api.mvc._
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
-import scala.concurrent.Future
-import Headers._
-
-/** Logs all requests, the time the app took to respond, and the status of the response */
-object LogRequestsFilter extends Filter {
```
Comment (jacobwinch) [inline membership-attribute-service/app/filters/LogRequestsFilter.scala]: Was this unused? Presumably we record all of this information in the Fastly logs anyway...

Comment (svillafe) [reply membership-attribute-service/app/filters/LogRequestsFilter.scala]: Yes, we ran a full-text search of this object in the codebase and we didn't find it.

File: membership-attribute-service/app/Global.scala [hunk 2]
```diff
@@ -1,23 +1,17 @@
...
 import scala.concurrent.Future
 
 
-object Global extends WithFilters(
-  CheckCacheHeadersFilter,
-  CSRFFilter(),
-  AddEC2InstanceHeader,
-  AddGuIdentityHeaders
-) {
+object Global extends GlobalSettings {
```
Comment (jacobwinch) [inline membership-attribute-service/app/Global.scala]: It looks like we should be aiming to remove this altogether in a future PR? https://www.playframework.com/documentation/2.6.x/Migration25#Deprecated-GlobalSettings

File: membership-attribute-service/app/filters/LogRequestsFilter.scala (hunk 1 above)
Comment (jacobwinch) [reply membership-attribute-service/app/filters/LogRequestsFilter.scala]: 👍

**Pull request merged**